package.json

```diff
@@ -4,7 +4,7 @@
     "node": "16.x",
     "npm": "8.x"
   },
-  "version": "3.0.1",
+  "version": "3.0.2-snapshot.3",
```

Comment:
I don't know this codebase well, but double checking to ensure the `snapshot` here is intended.

Reply:
It gets editted by `yarn version --prerelease --preid=snapshot`.  Perhaps this shouldnt be versioned with a snapshot build though.  @SeanLeRoy ?

Reply:
Its fine since it reflects reality